.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,23 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - main
+  # pull_request event is required only for autolabeler
+  pull_request:
+    # Only following types are handled by the action, but one can default to all as well
+    types: [opened, reopened, synchronize]
+  # pull_request_target event is required for autolabeler to support PRs from forks
+  pull_request_target:
```

Comment:
Initially I planned to request that we drop this `on` condition. However, it seems that the diceyness involves `pull_request_target` and a checkout.  There's no checkout, so /lgtm.